### ISSUE TITLE
Flash inoperable

### ISSUE THREAD

**Issue by Sauercrout** [dnschneid/crouton#2079]
Not sure if I'm in the right forum here, but here goes:

A couple days ago I performed the usual sudo startlxde to start up Ubuntu on my Chromebook, except I accidentally typed sudo startunity out of habit. (I was using Unity before and wanted to switch it up to a lighter DE, which I installed without much of a problem a while back) The result: a black screen. Because I couldn't do anything, I exited out to Chrome OS and tried again with 'sudo startlxde', which worked fine.

After this happened I wanted to check some Youtube vids and it occured to me Flash was acting up. I ran Synaptic update, no results. Manually apt-get update, it did find some things, but no result for Flash acting up. Removed flash, reinstalled through USC; apt-get purged it, even tried checking if Chromium would work with pepperflash-nonfree, and it stunningly showed the exact same behavior:
It loads at 0:00 and freezes. On the Twitch website (twitch.tv) it shows a similar behavior, freeze into crash 30 seconds later and an error report window comes up. Sometimes it plays a couple of seconds of choppy video, then freezes once more. Did the usual cleared cache, uninstall/reinstall add-ons, tried a different Firefox profile, to no avail. Because Chromium was a fresh installation, I'm led to believe it's something more serious outside the two browsers.

On Chrome OS (I believe that's the OS that the Chromebook comes with?) I can watch videos, no problem. They load instantaneously, showing no indication anything has changed on that end. There is no UpArrow icon in the tray bar signaling updates for Chrome OS.

When this freezing occurs, the CPU Resource Monitor doesn't show 100% use, instead it's either around 30% (normal for a medium quality stream or 480p Youtube video) or near idle. Htop in terminal mirrors this result. The browser itself will sometimes also spit out a generic "app is unresponsive" error when a Flash required-page is loaded.

I'm not 100% sure this is happening because of the accidental wrong target I used (unity instead of lxde) or if it's something else I'm overlooking, but I'm running on fumes here when it comes to solutions. I asked around, nobody seems to know what's up.

Does anyone know what I should do to fix this problem?

PS it would also seem the task bar is a blank bar for about 15-20 seconds until it finally loads - this didn't occur before.

tl;dr Adobe Flash freezes up on both Firefox and Chromium (pepperflash-nonfree) after an accidental wrong desktop environment was loaded. The two may or may not be related.

**Comment by Sauercrout** [comment]
Update: I solved this with a simple

sudo sh -e ~/Downloads/crouton -n trusty -u

Still not sure what caused it, but it seems to be working now. Sharing this in case others bump into the same problem. (HTML5 video was also affected, btw)
